src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -317,6 +318,7 @@ export const useSwapper = () => {
     if (precision) {
       formattedAmount = toBaseUnit(amount, precision)
     }
+    const feeAssetPrecision = feeAsset?.precision || 18 // ETH Fallback
```

Comment:
no fallback required here if we make `feeAsset` a concrete arg

Reply:
There has to be a fallback somewhere since the `sellAsset` initially is undefined initially sometimes. Moved it higher, so ETH is always passed unless the `sellAsset` has diff `feeAsset`.